evennia/objects/objects.py

```diff
@@ -1011,6 +1011,26 @@ def clear_contents(self):
                     obj.msg(_(string))
             obj.move_to(home, move_type="teleport")
 
+    @classmethod
+    def generate_default_locks(
+        cls, account: "DefaultAccount" = None, caller: "DefaultObject" = None, **kwargs
+    ):
+        """
+        Classmethod called during .create() to determine default locks for the object.
+
+        Args:
+            account (Account): Account to attribute this object to.
+            caller (DefaultObject): The object which is creating this one.
+            **kwargs: Arbitrary input.
+
+        Returns:
+            lockstring (str): A lockstring to use for this object.
+        """
+        if cls.lockstring:
+            account_id = account.id if account else -1
```

Comment:
This convention of setting id to `-1` is new. It seems like this could create some really hard-to-find bugs, if objects start being silently created with locks of `id(-1)` ... I would say we should error out in this situation; clearly something is wrong with the object creation or setup. The default should always return something reasonable out of the box.